setup.py

```diff
@@ -273,6 +273,8 @@ def run(self):
               "hypothesis >= 3.6.1",
               "treq",
               "towncrier",
+              "testtools",
+              "fixtures",
```

Comment:
Note that neither `testtools` nor `fixtures` is truly a new dependency.  These are both already transitive dependencies of Tahoe-LAFS.